client/src/shared/layouts/Container/index.tsx

```diff
@@ -2,10 +2,15 @@ import React from 'react';
 
 interface ContainerProps {
   children: any;
+  padding?: string;
 }
 
-const Container: React.FC<ContainerProps> = ({ children }: ContainerProps) => {
-  return <div className="container mx-auto px-20">{children}</div>;
+const Container: React.FC<ContainerProps> = ({ children, padding }: ContainerProps) => {
+  return <div className={`container mx-auto ${padding}`}>{children}</div>;
+};
+
+Container.defaultProps = {
+  padding: 'px-20'
```

Comment:
```suggestion
const Container: React.FC<ContainerProps> = ({ children, className = ' ' }: ContainerProps) => {
  return <div className={`container mx-auto ${className}`}>{children}</div>;
};
```

Padding is too specific, lets use more general name like `className` so we dont have to add a separate prop for each classNames in tailwind css